client/state/data-layer/wpcom/read/following/mine/new/index.js

```diff
@@ -18,52 +19,51 @@ import { bypassDataLayer } from 'state/data-layer/utils';
 
 import { registerHandlers } from 'state/data-layer/handler-registry';
 
-export function requestFollow( { dispatch }, action ) {
-	const {
-		payload: { feedUrl },
-	} = action;
+export function requestFollow( action ) {
+	const feedUrl = get( action, 'payload.feedUrl' );
 
-	dispatch(
-		http( {
-			method: 'POST',
-			path: '/read/following/mine/new',
-			apiVersion: '1.1',
-			body: {
-				url: feedUrl,
-				source: config( 'readerFollowingSource' ),
-			},
-			onSuccess: action,
-			onFailure: action,
-		} )
-	);
+	return http( {
+		method: 'POST',
+		path: '/read/following/mine/new',
+		apiVersion: '1.1',
+		body: {
+			url: feedUrl,
+			source: config( 'readerFollowingSource' ),
+		},
+		onSuccess: action,
+		onFailure: action,
```

Comment:
you can also leave these blank and pass `action` as a second parameter to `http` if you want

```js
http( { method, path, … }, action );
```

Reply:
This is great information. I already wondered and wanted to ask you about this 👍